src/cljs/org/broadinstitute/firecloud_ui/page/import_data.cljs

```diff
@@ -0,0 +1,36 @@
+(ns org.broadinstitute.firecloud-ui.page.import-data
+  (:require
+    [clojure.string]
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
+    [org.broadinstitute.firecloud-ui.utils :as utils :refer [rlog jslog cljslog ajax]]
+    ))
+
+
+(defn- submit-entities [workspaceName workspaceNamespace entityFile]
+  (utils/ajax-orch
+    (str "/workspaces/" workspaceNamespace "/" workspaceName "/importEntitiesJSON")
+    {:method :post
+     :encType "multipart/form-data"
+     :on-done :default
+     :data (utils/generate-form-data {:entities entityFile})}))
+
+
+(react/defc Page
+            {:render
```

Comment:
Preferences -> Editor -> Code Style -> Clojure -> General tab -> "Default to Only Indent" should be checked.